strapi/strapi
# [Front] Add warning message to confirm publish dialog

What does it do?
It shows the number of draft Relations of the selected entries in the Notification warning message.

How to test it?
Selected items in a content type with draft & publish enabled with draft relations on them
WHEN I click publish from the entry selection modal
THEN the confirm publish dialog shows a message warning me about the total number of draft relations

You can have a look at the mockups here https://www.figma.com/file/64cWvO8k1TWBDz39Kz1pZt/%5B%F0%9F%8F%97WIP%5D-Content-Studio---Bulk-publish?type=design&node-id=257-18371&t=cRlKutVykj15jpVf-0

ticket => CS-102
